api/provider/workos.go

```diff
@@ -0,0 +1,84 @@
+package provider
+
+import (
+	"context"
+	"strings"
+
+	"github.com/netlify/gotrue/conf"
+	"golang.org/x/oauth2"
+)
+
+const (
+	defaultWorkOSAPIBase = "https://api.workos.com"
```

Comment:
you can leave out the `https://` because gotrue will use https to make the redirect call